src/docs/product/sentry-basics/guides/getting-started/index.mdx

```diff
@@ -1,128 +1,133 @@
 
-Click on a team name to open the team settings to manage its associated members and projects.
-Under the `Members` tab add existing members to your team by clicking on `Add Member > [Member Name]`.
+![Create New Team](03.png)
 
-![Add Members](04.png)
+Click on a team name to open the team settings and manage its associated members and projects.
+Under the "Members" tab, add existing members to your team by clicking on "Add Member > [Member Name]".
 
-You can also invite multiple new (external) members to join your organization and team by clicking on `Add Member > Invite Member`
+You can also invite multiple new (external) members to join your organization and team by clicking on "Add Member > Invite Member".
 
 ![Invite Members](05.png)
 
-For more information see: [Organization Management](/product/accounts/membership/)
+Learn more about different user roles in [Organization Management](/product/accounts/membership/).
 
 ## 3. Configure Integrations
 
-Sentry integrates seamlessly into your existing workflows by providing out of the box integrations with your favorite apps and services. To enable and configure integrations, navigate to `Organization Settings > Integrations`.
-
-### 3.1. Alert Notifications
+Sentry integrates into your existing workflows by providing out-of-the-box integrations with widely-used apps and services. To enable and configure integrations, go to **Settings > Integrations**. There are several types of integrations available, but we recommend that you immediately set up integrations for:
+- [Alerting](#31-alert-notifications)
+- [Source code management](#32-source-code-management)
+- [Issue tracking](#33-issue-tracking)
 
-By default, Sentry will notify you about errors in your apps via **email**. To receive real-time notifications about your errors where you work, enable Sentry integrations with: [Slack](/product/integrations/notification-incidents/slack/), [PagerDuty](/product/integrations/notification-incidents/pagerduty/).
+### Alert Notifications {#31-alert-notifications}
 
-> ![PagerDuty Integration](slack_sentry_e2e_integration4.gif)
->
-> _Enabling alert notifications through the **Slack** integration._
+By default, Sentry will notify you about errors in your apps by email and [Slack](/product/integrations/notification-incidents/slack/), once you enable the integration. You can also enable team notifications in Slack so the right people in your organization always receive the alerts they need. In addition to Slack, Sentry offers several other [notification and incident integrations](/product/integrations/notification-incidents/), like [Microsoft Teams](/product/integrations/notification-incidents/msteams/) and [PagerDuty](/product/integrations/notification-incidents/pagerduty/), to help you stay informed about the errors you need to see.
```

Comment:
> By default, Sentry will notify you about errors in your apps by email and Slack, once you enable the integration. 

The three clauses here are a bit awkward. I'd put the `once you enable the integration` in parentheses.

Reply:
The clauses 
> so the right people in your organization always receive the alerts they need. 

and
> to help you stay informed about the errors you need to see.

are saying the same thing. Maybe we could say something about triaging issues?